### ISSUE TITLE
Compatibility Question

### ISSUE THREAD

**Issue by mahdi65** [tinyvision-ai-inc/UPduino-v3.0#24]
Are UPduino V3.0 and V3.1 compatible with this "[HM01B0-UPD-EVN](https://www.latticesemi.com/products/developmentboardsandkits/himaxhm01b0)"? If positive, is there any documentation available on that subject?

**Comment by vr2045** [comment]
Yes, it's actually part of the Himax kit that Lattice sells (manufactured by tinyVision 😉)
The docs are part of the Lattice kit, please refer the the Lattice website for details.
